.github/workflows/po4a-comment-commit.yaml

```
@@ -0,0 +1,82 @@
+name: Commit po4a changes
+
+on:
+  issue_comment:
+    types: [created]
+
+permissions:
+  contents: write
+  issues: read
+  pull-requests: write
+  id-token: 'write'
+  statuses: write
+
+env:
+  PO4A_VERSION: "0.71"
+  PO4A_GH_URL: https://github.com/mquinson/po4a/releases/download
+
+jobs:
+  commit-comment:
+    name: Get po4a check comment
+    if: (github.event.issue.pull_request && startsWith(github.event.comment.body, 'po4a commit'))
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: xt0rted/pull-request-comment-branch@v2
+        id: comment-branch
+
+      - name: Set up repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ steps.comment-branch.outputs.head_ref }}
+
+      - name: Check if user has write access
+        id: write_access
+        uses: prince-chrismc/check-actor-permissions-action@v3
+        with:
+          permission: write
+
+      - name: Install po4a
+        run: |
+          sudo apt-get install -y po4a  # required as standalone po4a is missing some deps
+          wget --quiet ${{ env.PO4A_GH_URL }}/v${{ env.PO4A_VERSION }}/po4a-${{ env.PO4A_VERSION}}.tar.gz  # sadly the version from apt isn't working
+          export PO4A_DIR="po4a-${{ env.PO4A_VERSION }}"
+          tar -xf ${PO4A_DIR}.tar.gz
+
+      - name: Run po4a
+        run: |
+          export PO4A_DIR="po4a-${{ env.PO4A_VERSION }}"
+          export PERL5LIB="${PO4A_DIR}/lib:${PERL5LIB}"
+          ./${PO4A_DIR}/po4a -v po4a.cfg
+
+      - name: Check if translations require updating
+        run: |
+          git status --porcelain translations
+          if [[ `git status --porcelain translations` ]]; then
```

```suggestion
          if git status --porcelain translations; then
```

Or may be the reverse, I don't know what you're trying to do here

---

I'm checking if the command outputs anything.

---

I guess the intent was to only create files.txt if `git status --porcelain translations` shows no output

---

Ah, sorry, duplicated line.

---

Yeah, fixed. That was a leftover from po4a PR check.